Automattic-Tracks-iOS/TracksService.m

```diff
@@ -264,6 +260,22 @@ - (void)reachabilityChanged:(NSNotification *)notification
     }
 }
 
+#pragma mark - Private methods
```

Comment:
This block is just a re-org of the methods so the Reachability-related methods are grouped together. No changes in content. See commit 2ae92a152759ef2f1c45068f38e933779fede720.